buildenv/jenkins/docker-slaves/x86/centos6.9/Dockerfile

```diff
@@ -186,11 +188,12 @@ RUN wget https://archive.apache.org/dist/ant/binaries/apache-ant-1.10.5-bin.zip
   && rm -rf ant-contrib
 
 # Install Cuda version 9.0
-RUN cd /usr/src/ \
-  && wget https://developer.nvidia.com/compute/cuda/9.0/Prod/local_installers/cuda_9.0.176_384.81_linux-run \
-  && chmod +x cuda_9.0.176_384.81_linux-run \
-  && /usr/src/cuda_9.0.176_384.81_linux-run --silent --toolkit --override \
-  && rm -f /usr/src/cuda_9.0.176_384.81_linux-run
+ENV CUDA_VERSION 9.0
+ENV CUDA_HOME    /usr/local/cuda-${CUDA_VERSION}
+
+RUN mkdir -p ${CUDA_HOME}/nvvm
+COPY --from=cuda-dev ${CUDA_HOME}/include      ${CUDA_HOME}/include
+COPY --from=cuda-dev ${CUDA_HOME}/nvvm/include ${CUDA_HOME}/nvvm/include
```

Comment:
I found out the hard way that references such as these depend upon the environment where `docker build` runs. To be sure the paths are as expected, they should be hard-coded.

Reply:
Hi Keith. Thanks for your advice! I will make the change.